app/lib/features/pins/widgets/pin_item.dart

```diff
@@ -82,22 +83,25 @@ class _PinItemState extends ConsumerState<PinItem> {
       );
     }
     content.add(
-      Expanded(
-        child: HtmlEditor(
-          editable: canEdit,
-          autoFocus: autoFocus,
-          content: widget.pin.contentText(),
-          footer: !canEdit ? const SizedBox() : null,
-          onCancel: () {},
-          onSave: (plain, htmlBody) async {
-            if (_formkey.currentState!.validate()) {
-              pinEditNotifier.setLink(_linkController.text);
-              pinEditNotifier.setPlainText(plain);
-              pinEditNotifier.setHtml(htmlBody);
-              final res = await pinEditNotifier.onSave(context);
-              debugPrint('PIN EDITED: $res');
-            }
-          },
+      Flexible(
```

Comment:
Love that it is inline, but I'm a bit puzzled by this. Does that mean we are always in the editor when we are looking at an individual Pin? That doesn't sound like a regular usecase though ... most of the time the person wants to look it up, not edit it. Editing features should probably be put behind some option-button->edit or something ...

Reply:
Hmm, that sounds sensible. I will change it into enabling/disabling edit mode 👍🏻 !